### DISCUSSION TITLE
bug empty releations [Jul 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2879] [Discord Threads]
> 🙋‍♂️ **lublak**  
## I have a strange behaviour: empty relations  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/999574615959547965/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **lublak**  
#### This is actually a "to many" relationship.  
#  
> 🙋‍♂️ **lublak**  
#### If i click on them (also empty):  
![image0](https://cdn.discordapp.com/attachments/999574616810987630/999577292076486726/unknown.png)  
#  
> 🙋‍♂️ **lublak**  
#### The PurchasingPages table:  
![image0](https://cdn.discordapp.com/attachments/999574616810987630/999577711011975198/unknown.png)  
#  
> 🙋‍♂️ **lublak**  
#### so the other way round it shows everything ok.  
#  
> 🙋‍♂️ **lublak**  
#### The ddls:   
####    
#### ```sql   
#### CREATE TABLE purchasing_pages (   
####     purchasing_id int8 NOT NULL,   
####     page_path varchar(255) NOT NULL,   
####     CONSTRAINT purchasing_pages_check_page_path CHECK (((page_path)::text ~ '^([a-zA-Z0-9\-_ ]+\/)+[a-zA-Z0-9\-_ ~]+\.[tT][iI][fF]'::text)),   
####     CONSTRAINT purchasing_pages_purchasing_id_page_path_key UNIQUE (purchasing_id, page_path)   
#### );   
####    
#### ALTER TABLE purchasing_pages ADD CONSTRAINT purchasing_pages_id_fkey FOREIGN KEY (purchasing_id) REFERENCES purchasing(id) ON UPDATE CASCADE ON DELETE RESTRICT;   
#### ```   
####    
#### ```sql   
#### CREATE TABLE purchasing (   
####     id int4 NOT NULL GENERATED BY DEFAULT AS IDENTITY,   
####     mandant_id int2 NOT NULL,   
####     company_code int4 NOT NULL,   
####     sequence_number int8 NOT NULL,   
####     "year" int2 NOT NULL,   
####     "month" int2 NOT NULL,   
####     CONSTRAINT purchasing_check_month CHECK (((month >= 1) AND (month <= 12))),   
####     CONSTRAINT purchasing_check_year CHECK (((year >= 2000) AND (year <= 2100))),   
####     CONSTRAINT purchasing_client_plant_sequence_number_key UNIQUE (mandant_id, company_code, sequence_number),   
####     CONSTRAINT purchasing_pkey PRIMARY KEY (id)   
#### );   
#### CREATE INDEX purchasing_id_idx ON purchasing USING btree (id);   
#### ALTER TABLE purchasing ADD CONSTRAINT purchasing_company_code_fk FOREIGN KEY (company_code) REFERENCES company_codes(id);   
#### ALTER TABLE purchasing ADD CONSTRAINT purchasing_mandant_id_fkey FOREIGN KEY (mandant_id) REFERENCES purchasing_mandant_names(id) NOT VALID;   
#### ```  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999574616810987630)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
